pallets/parachain-staking/src/lib.rs

```diff
@@ -3047,12 +3044,8 @@ pub mod pallet {
 			amount: BalanceOf<T>,
 		) -> DispatchResult {
 			let mut state = <CandidateInfo<T>>::get(&candidate).ok_or(Error::<T>::CandidateDNE)?;
-			let total_changed =
-				state.rm_delegation_if_exists::<T>(&candidate, delegator.clone(), amount)?;
+			state.rm_delegation_if_exists::<T>(&candidate, delegator.clone(), amount)?;
 			T::Currency::unreserve(&delegator, amount);
-			if state.is_active() && total_changed {
```

Comment:
Same comment here as before, what happens if its bottom?

Reply:
if its just bottom then `total_changed == false` so it would never enter this if branch